src/main/java/com/gmail/goosius/siegewar/utils/TownPeacefulnessUtil.java

```diff
@@ -62,20 +62,24 @@ public static void updateTownPeacefulnessCounters(Town town) {
 			return;
 		}
 		TownMetaDataController.setPeacefulnessChangeDays(town, 0);
+
 		town.setNeutral(!town.isNeutral());
+		
+		if (town.isNeutral())
+			town.setAdminDisabledPVP(true);
+		else
+			town.setAdminDisabledPVP(false);
```

Comment:
This is the only thing which needs to be changed in the TownPeacefulnessUtil.java class. Please revert the rest of the changes.